mesa/experimental/jupyter_viz.py

```diff
@@ -83,83 +116,98 @@ def handle_change_model_params(name: str, value: any):
                 make_plot(model, measure)
 
 
-@solara.component
-def ModelController(
-    model, play_interval, current_step, set_current_step, reset_counter
-):
+def BaseControls(play_interval, on_step, on_reset):
```

Comment:
Why is there no `@solara.component` here?

Reply:
Forgot it